io/include/pcl/io/file_io.h

```diff
@@ -246,37 +247,18 @@ namespace pcl
     else
       stream << boost::numeric_cast<Type>(value);
   }
-  template <> inline void
-  copyValueString<int8_t> (const pcl::PCLPointCloud2 &cloud,
-                           const unsigned int point_index, 
-                           const int point_size, 
-                           const unsigned int field_idx, 
-                           const unsigned int fields_count, 
-                           std::ostream &stream)
-  {
-    int8_t value;
-    memcpy (&value, &cloud.data[point_index * point_size + cloud.fields[field_idx].offset + fields_count * sizeof (int8_t)], sizeof (int8_t));
-    if (std::isnan (value))
-      stream << "nan";
-    else
-      // Numeric cast doesn't give us what we want for int8_t
-      stream << boost::numeric_cast<int>(value);
```

Comment:
I'm a little concerned over the removal of this specialization. When a value is of `int8_t`, printing to streams usually would dump a `char`, which is not what is desired here. Example:
```cpp
int8_t var = 0;
std::cout << var << std::endl;
```
This will either print a funky character or nothing at all, when when a type is `int8_t`, the user is really interested in seing a numerical value. Hence the cast and this particular specialization. By removing it entirely, we're gonna have the same problem again.

Reply:
Good point, I agree that we should keep the byte specializations.

Reply:
Ah well, now I anderstand the comment `Numeric cast doesn't give us what we want for int8_t`. I readded these functions and changed comment, so next know why this type cast is necessary there.

But general `boost::numeric_cast` in general integral and floating point specialization is senseless, or? As far as I unterstand documentation this cast only makes sense in case types before and after cast are different.

Reply:
I removed unnecessary `boost::numeric_cast`. In case you see any reason for keeping this cast, tell me.